### DISCUSSION TITLE
Problem returning data from firebase firestore

### DISCUSSION THREAD

**Discussion by burningman2k20** [proyecto26/RestClient#240] [General]
[Serializable]
public class RestUserData{
    public string Id;
    public RestCharacterData[] characters;

}
trying to return this data using the RestClient.Get<RestUserData> method. the data gets returned from firebase firestore. the debug log says it gets the data but it never gets assigned to the callback because its null. ive tried everything i can think of but nothing gets returned.

**Comment by maifeeulasad** [comment]
Dear,

Can you create a small toy project, and link it here? Then we can try our best to resolve this issue.

**Reply by burningman2k20** [reply]
I was finally able to get it to return the data..has to borrow code from another GitHub project called firestorm with a class for the firestorm rest API to be put into. Problem. Im having now is actually looping thru that data and being able to take what I need. Because the data from.firebase is not standard JSON format it's taking some extra fiddling around to sift thru it. I didn't realize that it wouldn't do that stuff automatically. Any help for sifting thru the firestorm data would certainly help. the original firestormr code to sift thru the data does not work anymore
